yt_dlp/extractor/prx.py

```diff
@@ -0,0 +1,449 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import itertools
+from .common import InfoExtractor, SearchInfoExtractor
+from ..utils import (
+    urljoin,
+    traverse_obj,
+    int_or_none,
+    mimetype2ext,
+    clean_html,
+    url_or_none,
+    unified_timestamp,
+    str_or_none,
+)
+
+
+class PRXBaseIE(InfoExtractor):
+    PRX_BASE_URL_RE = r'https?://(?:(?:beta|listen)\.)?prx.org/%s'
+
+    def _call_api(self, item_id, path, query=None, fatal=True, note='Downloading CMS API JSON'):
+        return self._download_json(
+            urljoin('https://cms.prx.org/api/v1/', path), item_id, query=query, fatal=fatal, note=note)
+
+    @staticmethod
+    def _extract_embedded_data(response):
+        return traverse_obj(response, '_embedded', expected_type=dict, default={})
+
+    @classmethod
+    def _get_prx_embed_response(cls, response, section):
+        return cls._extract_embedded_data(response).get(f'prx:{section}')
```

Comment:
```suggestion
    def _get_prx_embed_response(response, section):
        return traverse_obj(response, ('_embedded', f'prx:{section}'))
```
I noticed _extract_embedded_data isn't used anywhere else. This probly doesnt even need to be a function though